paper_template/Makefile

```diff
@@ -0,0 +1,25 @@
+FILE=report
+
+all:
+	pdflatex ${FILE}
+	bibtex ${FILE}
+	pdflatex ${FILE}
+	pdflatex ${FILE}
+
+real-clean: clean
+	rm -rf *.pdf
+
+clean:
+	rm -rf *~ *.aux *.bbl *.dvi *.log *.out *.blg *.toc *.fdb_latexmk *.fls *.tdo *.bcf
+
+view:
+	open ${FILE}.pdf
```

Comment:
This will only work on macOS. Could you adapt the Makefile from cloudmesh/classes. It switches based on the output of `uname` and uses xdg-open for Linux.